ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -46,6 +46,9 @@ ref:
   - name: OPENSHIFT_INSTALL_AWS_PUBLIC_ONLY
     default: ""
     documentation: "Whether to use only public subnets for AWS. Implies no NAT Gateways. Requires a VPC to be configured prior to install."
+  - name: ENABLE_EFA_PG_INSTANCE
```

Comment:
This feature is only available on AWS, right? If yes, suggest to rename the env to `ENABLE_AWS
_EFA_PG_INSTANCE`

Reply:
done